Feature/netskope enhancement (#36351)
demisto/content: Packs/Netskope/Integrations/NetskopeAPIv2/README.md

| Netskope.Alert.skip_geoip_lookup         | String   | A flag indicating whether GeoIP lookup should be skipped.                                                                                                                                                                                                                                                                                                                     |
| Netskope.Alert.src_epoch_now             | Number   | A numeric value representing the source epoch.                                                                                                                                                                                                                                                                                                                                |

> | Netskope.Alert.access_method             | String   | Cloud app traffic can be steered to the Netskope cloud using different deployment methods such as Client (Netskope Client), Secure Forwarder etc. Administrators can also upload firewall and/or proxy logs for log analytics. This field shows the actual access method that triggered the event. For log uploads this shows the actual log type such as PAN, Websense, etc. |

Reviewer: ```suggestion
| Netskope.Alert.access_method             | String   | Cloud app traffic can be steered to the Netskope cloud using different deployment methods such as Client (Netskope Client), Secure Forwarder etc. Administrators can also upload firewall and/or proxy logs for log analytics. This field shows the actual access method that triggered the event. For log uploads, this shows the actual log type such as PAN, Websense, etc. |
```

| Netskope.Alert.device_classification     | String   | Designation of the device as determined by the Netskope Client as to whether the device is managed or not.                                                                                                                                                                                                                                                                    |
| Netskope.Alert.domain                    | String   | Domain value. This will hold the host header value or SNI or extracted from an absolute URI.                                                                                                                                                                                                                                                                                  |

> | Netskope.Alert.dst_country               | String   | Application’s two-letter country code as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                    |
> | Netskope.Alert.dst_latitude              | Number   | Latitude of the application as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                              |
> | Netskope.Alert.dst_location              | String   | Application’s city as determined by maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                       |
> | Netskope.Alert.dst_longitude             | Number   | Longitude of the application as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                             |
> | Netskope.Alert.dst_region                | String   | Application’s state or region as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                            |
> | Netskope.Alert.dst_timezone              | String   | Destination timezone.                                                                                                                                                                                                                                                                                                                                                         |
> | Netskope.Alert.dst_zipcode               | String   | Application’s zip code as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| Netskope.Alert.dst_country               | String   | Application’s two-letter country code as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                    |
| Netskope.Alert.dst_latitude              | Number   | Latitude of the application as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                              |
| Netskope.Alert.dst_location              | String   | Application’s city as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                       |
| Netskope.Alert.dst_longitude             | Number   | Longitude of the application as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                             |
| Netskope.Alert.dst_region                | String   | Application’s state or region as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                            |
| Netskope.Alert.dst_timezone              | String   | Destination timezone.                                                                                                                                                                                                                                                                                                                                                         |
| Netskope.Alert.dst_zipcode               | String   | Application’s zip code as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                   |
```

| Netskope.Alert.severity                  | String   | Severity used by watchlist and malware alerts.                                                                                                                                                                                                                                                                                                                                |
| Netskope.Alert.site                      | String   | For traffic_type = CloudApp, site = app, and for traffic_type = Web, it will be the second-level domain name + top-level domain name. For example, in “www.cnn.com”, it is “cnn.com”.                                                                                                                                                                                         |

> | Netskope.Alert.src_country               | String   | User’s country’s two-letter country code as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                 |
> | Netskope.Alert.src_latitude              | Number   | Latitude of the user as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                     |
> | Netskope.Alert.src_location              | String   | User’s city as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                              |
> | Netskope.Alert.src_longitude             | Number   | Longitude of the user as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                    |
> | Netskope.Alert.src_region                | String   | Source state or region as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                   |
> | Netskope.Alert.src_time                  | Date     | A timestamp associated with the source or event.                                                                                                                                                                                                                                                                                                                              |
> | Netskope.Alert.src_timezone              | String   | Source timezone. Shows the long-format timezone designation.                                                                                                                                                                                                                                                                                                                  |
> | Netskope.Alert.src_zipcode               | String   | Source zip code as determined by Maxmind or IP2Location Geodatabase.                                                                                                                                                                                                                                                                                                          |

Reviewer: ```suggestion
| Netskope.Alert.src_country               | String   | User’s country’s two-letter country code as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                 |
| Netskope.Alert.src_latitude              | Number   | Latitude of the user as determined by MMaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                     |
| Netskope.Alert.src_location              | String   | User’s city as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                              |
| Netskope.Alert.src_longitude             | Number   | Longitude of the user as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                    |
| Netskope.Alert.src_region                | String   | Source state or region as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                   |
| Netskope.Alert.src_time                  | Date     | A timestamp associated with the source or event.                                                                                                                                                                                                                                                                                                                              |
| Netskope.Alert.src_timezone              | String   | Source timezone. Shows the long-format timezone designation.                                                                                                                                                                                                                                                                                                                  |
| Netskope.Alert.src_zipcode               | String   | Source zip code as determined by MaxMind or IP2Location Geolocation database.                                                                                                                                                                                                                                                                                                          |
```

| Netskope.Alert.nsp_retrans_back          | Number   | Retransmission information for NSP on the back end.                                                                                                                                                                                                                                                                                                                           |
| Netskope.Alert.nsp_retrans_front         | Number   | Retransmission information for NSP on the front end.                                                                                                                                                                                                                                                                                                                          |

> | Netskope.Alert.nsp_rtt_back              | Number   | Round-trip time information for NSP on the back end.                                                                                                                                                                                                                                                                                                                          |
> | Netskope.Alert.nsp_rtt_front             | Number   | Round-trip time information for NSP on the front end.                                                                                                                                                                                                                                                                                                                         |

Reviewer: ```suggestion
| Netskope.Alert.nsp_rtt_back              | Number   | Roundtrip time information for NSP on the back end.                                                                                                                                                                                                                                                                                                                          |
| Netskope.Alert.nsp_rtt_front             | Number   | Roundtrip time information for NSP on the front end.                                                                                                                                                                                                                                                                                                                         |
```

#### Input

| **Argument Name** | **Description**                                                                                                                                                                                                                                  | **Required** |
| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------ |
| url_list_id       | The URL list ID to update (use netskope-url-list-list command to get URL list ID).                                                                                                                                                               | Required     |
| name              | The updated URL list name.                                                                                                                                                                                                                       | Required     |

> | urls              | The updated URL list items (For Exact - Enter URLs like _.example.com, or IP addresses, separated by new line. For Regex - Enter URLs like ^client[0-9]\\.google\\.com , ^app\\.slack\\.com/._/netskope, or ^google.com, separated by new line). | Required     |

Reviewer: ```suggestion
| urls              | The updated URL list items (For Exact - Enter URLs like _.example.com, or IP addresses, separated by a new line. For Regex - Enter URLs like ^client[0-9]\\.google\\.com , ^app\\.slack\\.com/._/netskope, or ^google.com, separated by a new line). | Required     |
```

#### Input

| **Argument Name** | **Description**                                                                                                                                                                                                                          | **Required** |
| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
| name              | The unique name for the URL list.                                                                                                                                                                                                        | Required     |

> | urls              | The URL list items (For Exact - Enter URLs like _.example.com, or IP addresses, separated by new line. For Regex - Enter URLs like ^client[0-9]\\.google\\.com , ^app\\.slack\\.com/._/netskope, or ^google.com, separated by new line). | Required     |

Reviewer: ```suggestion
| urls              | The URL list items (For Exact - Enter URLs like _.example.com, or IP addresses, separated by a new line. For Regex - Enter URLs like ^client[0-9]\\.google\\.com , ^app\\.slack\\.com/._/netskope, or ^google.com, separated by a new line). | Required     |
```

#### Input

| **Argument Name** | **Description**                                                                                                                                                                                                                                  | **Required** |
| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------ |
| url_list_id       | The URL list ID to update (use netskope-url-list-list command to get URL list ID).                                                                                                                                                               | Required     |

> | urls              | The updated URL list items (For Exact - Enter URLs like _.example.com, or IP addresses, separated by new line. For Regex - Enter URLs like ^client[0-9]\\.google\\.com , ^app\\.slack\\.com/._/netskope, or ^google.com, separated by new line). | Required     |

Reviewer: ```suggestion
| urls              | The updated URL list items (For Exact - Enter URLs like _.example.com, or IP addresses, separated by a new line. For Regex - Enter URLs like ^client[0-9]\\.google\\.com , ^app\\.slack\\.com/._/netskope, or ^google.com, separated by a new line). | Required     |
```
